docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -41,11 +41,35 @@ sdlManager.systemCapabilityManager.updateCapabilityType(.phoneCall) { (error, sy
 
 @![android,javaSE,javaEE]
 ```java
-HMICapabilities hmiCapabilities = (HMICapabilities)sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.HMI);
-if (hmiCapabilities.isPhoneCallAvailable()) {
-    // DialNumber supported
-} else {
-    // DialNumber is not supported
+sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.PHONE_CALL, new OnSystemCapabilityListener() {
```

Comment:
I updated the code snippet. Not sure what you mean about `getCapability()` not working since there is a fallback check for `SystemCapabilityType.PHONE_CALL`.

Reply:
See above comment, does that explain better?